src/app/resources/patreon.py

```diff
@@ -38,12 +38,18 @@ def get(self):
         else:
             return redirect(f"{cfg.API['website-url']}/linkpatreon?error=unknown")
 
-        return redirect(f"{cfg.API['website-url']}/linkpatreon?error=unknown")
 
+class PatreonLinkSchema(Schema):
+    ckey = fields.String()
+    patreon_id = fields.String()
 
-class LinkedPatreonListResource(Resource):
-    def get(self):
-        if request.args.get("pass") == cfg.PRIVATE["api_passwd"]:
+
+class LinkedPatreonListResource(MethodResource):
+    @marshal_with(PatreonLinkSchema(many=True))
+    @use_kwargs(APIPasswordRequiredSchema)
+    @doc(description="Get a list of linked ckey-Patreon accounts.")
+    def get(self, **kwargs):
+        if kwargs["api_pass"] == cfg.PRIVATE["api_passwd"]:
```

Comment:
```suggestion
        if kwargs["pass"] == cfg.PRIVATE["api_passwd"]:
```

Reply:
Pass is a required arg so it will always be defined.